src/types/konflux-public-info.ts

```diff
@@ -45,6 +45,7 @@ export type KonfluxInstanceVisibilityType =
 
 export type KonfluxPublicInfo = {
   environment?: string;
+  status_page_url?: string;
```

Comment:
Please use camelCase naming convention here
```suggestion
  statusPageUrl?: string;
```